### ISSUE TITLE
Bug: BreakPanels display newline as an empty character

### ISSUE THREAD

**Issue by JesFo** [TomaszRewak/C-sharp-console-multiplexer#32]
Hello.
I have an issue, where drawing over a `Border` control with a `BreakPanel ` results in every newline character `\n` overdrawn as an empty character over the underlying control, even when using an `Overlay `control.
For example:
```
Content = new BreakPanel
{
    Content = new TextBlock {Text = "┬\n│\n│\nTest\n[Value]" }
}
```
Drawn over a border looks like this:
![fig0](https://github.com/TomaszRewak/C-sharp-console-gui-framework/assets/90457978/ee921067-6a60-4c66-9a1a-c677ddc94b95)

**Comment by TomaszRewak** [comment]
Hey, thanks for reporting!

I was able to reproduce the issue.

The minimum required code for reproducing it is:
```csharp
using ConsoleGUI;
using ConsoleGUI.Controls;

class Program
{
	public static void Main()
	{
		ConsoleManager.Setup();

		ConsoleManager.Content = new Overlay
		{
			BottomContent = new BreakPanel
			{
				Content = new TextBlock
				{
					Text = "XXXXXXXXX\nXXXXXXXXX\nXXXXXXXXX\nXXXXXXXXX"
				}
			},
			TopContent = new BreakPanel
			{
				Content = new TextBlock
				{
					Text = "ooooo\nooo\noo"
				}
			}
		};

		while (true)
		{
			ConsoleManager.AdjustBufferSize();
		}
	}
}
```

Resulting in this:

![image](https://github.com/TomaszRewak/C-sharp-console-gui-framework/assets/5797540/b32e0f6d-326d-4d33-bb20-15537f0cfc2b)

Will take a look at how can that be fixed.

**Comment by TomaszRewak** [comment]
Should be fixed in the newly released version `1.4.2`.
Please let me know if it works for you.

**Comment by JesFo** [comment]
It works now. Thank you.
